### DISCUSSION TITLE
Adding audit comment to policy

### DISCUSSION THREAD

**Discussion by jbhoorasingh** [PaloAltoNetworks/pan-os-python#320] [Q&A]
Can someone point me to documentation on how to add an audit comment to a policy using pan-os-pyhton?

**Comment by shinmog** [comment]
This isn't in place yet - here is the github issue asking for this:  #272

**Reply by shinmog** [reply]
This was fixed here:  #323

**Reply by RyanMBess** [reply]
First, thanks so much for creating this utility.  I need some help.
1. Can someone please show some code for how to add a comment to an existing rule?
2. Can someone show some code for how to create a new rule and add comments to it.  

When we make new rules or changes to existing, we are both policy required AND the palos will not let us commit the change without a comment.  

Any help is much appreciated.

**Comment by shinmog** [comment]
https://github.com/PaloAltoNetworks/pan-os-python/issues/396#issuecomment-1007031023
